target/linux/realtek/files-5.10/drivers/clk/realtek/clk-rtl83xx.h

```diff
@@ -0,0 +1,87 @@
+// SPDX-License-Identifier: GPL-2.0-only
+/*
+ * Realtek RTL83XX clock headers
+ * Copyright (C) 2022 Markus Stockhausen
+ */
+
+/*
+ * Several stuff
+ */
+
+#define RTL_SRAM_MARKER				(0x5EAF00D5)
+#define RTL_SRAM_BASE				(0x9F000000)
```

Comment:
is this the same on all realtek's? even so, the file is called RTL83xxx, and you only have RTL83[89]x stuff here, so
```suggestion
 #define RTL83XX_SRAM_BASE				(0x9F000000) 
```
indentation is a bit off, but that might be github's rendering .... (spaces are also ok for _indentation_

Reply:
The line marker for patches can push a tab character into the next column.

Reply:
I don't this is it though; I think github renders 1 tab as 4 spaces generally. There's been quite a few bugs and issues on that topic.

Reply:
You're right, sorry for the noise. That's why I tend to only spaces for indentation on my own code. No hassle with tab widths.

Reply:
hah, I think tabs are far superior for _indentation_; it holds meaning. spaces are perfect for indentation however. using both (tabs for indentation, spaces for alignment0 works great :)

Don't worry though; spaces for indentation AND alignment have plenty of other issues :D but lets not get ourselves distracted on that :p